handlers/stock.js

```diff
@@ -0,0 +1,176 @@
+            LOG.error(error);
+        }
+    })
+
+    // Verify all users have an account and create new ones for those that don't
+    channel.guild.members.forEach(member => {
+        getStock(tableSvc, member.user.username, function (callback) {
+            if (typeof callback !== 'number') {
+                LOG.info('Setting up new $chreef $tock account for ' + member.user.username);
+                updateUser(tableSvc, member.user.username, STARTINGSTOCK);
+            }
+        })
+    });
+
+    if (splitParams[0] === 'help') {
+        channel.send('Command Usage:' + '\n' +
+            '.reset - Resets you back to 1000$$' + '\n' +
+            '.balance - Displays your balance' + '\n' +
+            '.allbalances - Displays everyone\'s balance' + '\n' +
+            '.send [username] [amount] - sends user amount of $$' + '\n');
+    }
+
+    // Reset command to go back to starting stock
+    if (splitParams[0] === 'reset') {
+        updateUser(tableSvc, message.author.username, STARTINGSTOCK);
+    }
+
+    // Get the stock for that user
+    if (splitParams[0] === 'balance') {
+        getStock(tableSvc, message.author.username, function (callback) {
+            channel.send(`${message.author.username}\'s balance is ${callback}$$`);
+        });
+    }
+
+    // Show the stock for all users
+    if (splitParams[0] === 'allbalances') {
+        showAllStock(tableSvc, channel);
+    }
+
+    // Send stock to another user if they have an account
+    if (splitParams[0] === 'send') {
+        const recipient = splitParams[1];
+        const sendAmount = parseInt(splitParams[2]);
+
+        if (recipient !== message.author.username) {
+            getStock(tableSvc, recipient, function (receiverStock) {
+                if (typeof receiverStock === 'number') {
+                    getStock(tableSvc, message.author.username, function (senderStock) {
+                        if ((senderStock - sendAmount) > 0 && typeof senderStock === 'number') {
+
+                            const newSenderTotal = senderStock - sendAmount;
+                            const newReceiverTotal = receiverStock + sendAmount;
+                            let transactionString = 'Transaction Complete:' + '\n';
+
+                            updateUser(tableSvc, message.author.username, newSenderTotal);
+                            updateUser(tableSvc, recipient, newReceiverTotal);
+
+                            transactionString += `${message.author.username}: ${senderStock} - ${sendAmount} = ${newSenderTotal}\n
+                                                  ${recipient}: ${receiverStock} + ${sendAmount} = ${newReceiverTotal}`
+
+                            LOG.info(transactionString);
+                            channel.send(transactionString);
+                        }
+                        else {
+                            channel.send('Not enough stock to make the transaction.')
+                        }
+                    })
```

Comment:
missing ;

Reply:
for all of these comments -these arent syntactically necessary right? This is just for cleanliness / consistency?

Reply:
if you only have a single statement in a block, like you do here, i think it works fine

but it is _technically_ incorrect to not have semicolons

i'm gonna get a eslint/jshint thing setup to let people know about of these issues on build/run, so we can keep our style consistent